src/services/system/hooks/verifyPayload.js

```diff
@@ -2,7 +2,8 @@ const { Forbidden } = require('../../../errors');
 
 module.exports = async (context) => {
 	const currentSystem = await context.app.service('systems').get(context.id);
-	if (currentSystem.type === 'ldap' && currentSystem.ldapConfig && currentSystem.ldapConfig.provider === 'iserv-idm') {
+
+	if (!currentSystem.ldapConfig || currentSystem.ldapConfig.provider !== 'general') {
```

Comment:
Didnt we wanted to use a nest service for validation here?
The advantage would be to have the logic at one place.

I didnt mentioned at in the client PR, but maybe we should introduce a feature toggle for it. What do you think?

Reply:
I don't like the idea of introducing a lot of nest services to feathers and we should only do it if there is no other option or if we have an extended amount of logic that should better be handled in typescript. I also think the effort of introducing a nest service to feathers, in relation to the one line validation, is too high. I would rather keep it simple.

Reply:
I don't entirely agree with that.

I think the argument for logic in one place weighs more than more services. You don't need a new service for everything, I think it's progress if at least the logic ends up in a nest service.

What do you think @alweber-cap

Reply:
I agree with @arnegns here. 

While it takes some additional effort to call the nest service, having the logic in one place is more beneficial and easier to maintain in the long run. Feathers will stay with us for quite some time, but moving more and more logic out of it is the right step for the whole software in my opinion.

Besides, if this logic needs to be changed again, chances are high that it will be changed only in Nest and forgotten here, which will lead to errors again, possibly undetected until quite some time in production